docs/content/manual/pre-release/node/node-deletion.md

```diff
@@ -11,22 +11,26 @@ title: Test node deletion
 ### Case 2: Delete nodes when there are running volumes:
 1. Deploy Longhorn.
 2. For each node that will be deleted later: 
-    1. create and attach 4 volumes:
+    1. Create and attach 5 volumes:
         1. The 1st volume contains 1 replica only. Both the engines and the replicas are on the pre-delete node. (Attached to the pre-delete node.)
         2. The 2nd volume contains 1 replica only. The engine is on the pre-delete node and the replica is on another node. (Attached to the pre-delete node.)
-        3. The 3rd and the 4th volume contain 3 replicas. Both volumes are attached to a node except for the pre-delete node.
+        3. The 3rd volume contains 1 replica only. The replica is on the pre-delete node and the engine is on another node. (Attached to a node except for the pre-delete node.)
+        4. The 4th volume contain 3 replicas. Attached to a node except for the pre-delete node.
+        5. The 5th volume contain 3 replicas. Attached to a node except for the pre-delete node.
     2. Write some data to all volumes.
 3. Delete multiple nodes in the cluster simultaneously. Make sure the corresponding Kubernetes node object is deleted. --> The related Longhorn node objects will be cleaned up immediately, too.
 4. For each deleted node:
     1. Verify the volume Health state  -->
-        1. The 1st volume should become `Faulted`.
+        1. The 1st volume should keep `Unknown`.
         2. The 2nd volume should keep `Unknown`.
-        3. The 3rd and the 4th volume should become `Degraded`.
-    2. Delete the 1st volume. --> The volume can be deleted.
+        3. The 3rd volume should become `Faulted`.
+        4. The 4th and 5th volume should become `Degraded`.
+    2. Detach then reattach the 1st volume. --> The volume can not be attached.
     3. Detach then reattach the 2nd volume. --> The volume works fine and the data is correct.
```

Comment:
What's the status for the 2? the volume will be in `attaching`?

Reply:
This volume can detach success, after that if I try to attach this volume to another node,  attach will faie and volume kept in detach state
```
unable to attach volume vol1 to ip-172-31-82-34: cannot attach volume vol1 because the engine image longhornio/longhorn-engine:v1.4.1-rc2 is not deployed on at least one of the the replicas' nodes or the node that the volume is going to attach to
```

Reply:
I see, then this is expected because the replica was on the deleted node. However, the message looks misleading, because it should be the only replica is failed on the deleted node, so there is no available replica instead.

[/datastore/longhorn.go#L1489-L1494](https://github.com/longhorn/longhorn-manager/blob/423513ad5838695c0b481a1c5309a0ecb73abbe9/datastore/longhorn.go#L1489-L1494)
```go
	for _, r := range replicas {
		isReady, err := s.CheckEngineImageReadiness(image, r.Spec.NodeID)
		if err != nil || isReady {
			return isReady, err
		}
	}
```